src/RouteResolver.tsx

```diff
@@ -86,44 +100,47 @@ export const generateSynapseObjectHelper = (synapseConfig: SynapseConfig) => {
   }
 }
 
-export const generateSynapseObject = (
```

Comment:
Renamed and now called through JSX since this is a function that unconditionally renders a React component